src/main/java/controller/user/AllUsersServlet.java

```diff
@@ -0,0 +1,27 @@
+package controller.user;
+
+import factory.UserServiceFactory;
+import model.User;
+import service.user.UserService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+@WebServlet(value = "/admin/users", loadOnStartup = 2)
```

Comment:
do we need loadOnStartup = 2 here? if yes, write why

Reply:
fixed